src/client/components/getAllAdminProducts.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Carousel from '../components/carousel';
+import adminPanelText from '../assets/local/adminPanel.json';
+import DeleteIcon from '../assets/images/DeleteIcon.png'
+import PenIcon from '../assets/images/PenIcon.svg';
+
+function GetAllAdminProducts (props) {
+    return props.db.map( (product) => {
+        const images = props.GetProductImages(product, 'small');
+        return (
+            <div key={product.uuid} className='product height'>
+                <div className='product-image'>                                      
+                    <Carousel images={images}/>                                  
+                </div>
+                <div className='product-information'>
+                    <p>{adminPanelText.uuid}: {product.uuid}</p>                 
+                    <p>{adminPanelText.name}: {product.name}</p>
+                    <p>{adminPanelText.price}: ${product.price}</p>                                        
+                    <p>{adminPanelText.description_short}: {product.description_short}</p>   
+                    <p>{adminPanelText.description_full}: {product.description_full}</p>
+                    <p>{adminPanelText.producer}: {product.producer}</p>   
+                    <p>{adminPanelText.amount}: {product.amount}</p>   
+                    <p>{adminPanelText.language}: {product.language}</p>
+                    <p>{adminPanelText.date}: {product.date}</p>
+                    <p>{adminPanelText.platform}: {product.platform}</p>
+                </div>
+                <div className='product-add-to-basket'>
+                    <button className="remove-product buy-button" onClick={
+                        () => {
+                            props.deleteProduct(product.uuid)                            
+                        }
+                    }><img className='basket-icon' src={DeleteIcon} /></button>
+                    <button className='change-button' onClick={
+                        () => {                                                                           
```

Comment:
@VRatick
Please use :

```
onClick= {() => {
//function body
}}
``` 
instead of your current format for arrow functions ( every time you use arrow function )